tests/Makefile

```diff
@@ -58,7 +64,7 @@ all: framework
 
 build:
 	@${ECHOE} ${GREEN}Build [$@] ${NC}
-	@make -C ../ test TARGET=$(TARGET)
+	@make -C ../ test TARGET=$(TARGET) DGTEST=${DGTEST}
```

Comment:
`@make -C ../ test TARGET=$(TARGET) DGTEST=${DGTEST}`

therefore would be

`@make -C ../ test TARGET=$(TARGET) VARIANT=${VARIANT}`

No-one has any idea what sort of framework we're using, today it's `CUNIT` & `GTEST` tomorrow it maybe `CPPTEST`